modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -348,14 +288,179 @@ processed.
 As said, you can wrap that command in a Cron job so it’s run every 5
 minutes for example.
 
+=== WND Listener Configuration
+
+Create a service following the instructions below that checks the share for changes:
+
+* For each WND mount point distinguished by a SERVER - SHARE pair,
+** place one copy of a file
+with following content under +
+`/etc/systemd/system/owncloud-wnd-listen-SERVER-SHARE.service`
+** replacing the all upper case words `SERVER`, `SHARE`, `USER` and `PASSWORD`
+** in both, the **filename** and in the **contents** below with their respective values. +
+Take care to also adjust the paths in `WorkingDirectory` and `ExecStart`
+according to your installation.
++
+....
+[Unit]
+Description=ownCloud WND Listener for SERVER SHARE
+After=syslog.target
+After=network.target
+Requires=apache2.service
+[Service]
+User=www-data
+Group=www-data
+WorkingDirectory=/var/www/owncloud
+ExecStart=/usr/bin/php ./occ wnd:listen -vvv SERVER SHARE USER PASSWORD
+Type=simple
+StandardOutput=journal
+StandardError=journal
+SyslogIdentifier=%n
+KillMode=process
+RestartSec=1
+Restart=always
+....
+
+* Run once for each created file the following commands:
++
+[source,console]
+----
+sudo systemctl enable owncloud-wnd-listen-SERVER-SHARE.service
+sudo systemctl start  owncloud-wnd-listen-SERVER-SHARE.service
+----
+
+=== WND Process Queue Configuration
+
+Create or add a `crontab` file in `/etc/cron.d/oc-wnd-process-queue`.
+
+* Make a `crontab` entry to run a script iterating over all `SERVER SHARE` pairs with
+an appropriate `occ wnd:process-queue` command. The commands must be **strictly sequential**.
+This can be done by using `flock -n` and tuning the `-c` parameter of `occ wnd:process-queue`
+
+....
+* * * * *     sudo -u www-data /usr/bin/php /var/www/owncloud/occ wnd:process-queue <HOST> <SHARE>
+....
+
+
+=== Serializer
+
+Parallel runs of `wnd:process-queue` might lead to a user lockout.
```

Comment:
I'd skip this paragrapth. If the process-queue is set with a cron job, it's extremely weird that a race condition in the specific point that would trigger this problem will happen. It's weird it happens if 2 processes start at the same time, so it's even more weird if it happens if the second process starts with more than 1 minute of delay.

The main reason to serialize the execution is to ensure a predictable order of the notifications.

Reply:
from where to where do you want the text to be omitted?

Reply:
@jvillafanez

Reply:
```
Parallel runs of `wnd:process-queue` might lead to a user lockout.
The reason for this is that several `wnd:process-queue` might use the same wrong password because it hasn't been updated by the time they fetch it.
```

Reply:
removed